packages/component-library-vue/src/UtrechtSelect.vue

```diff
@@ -1,17 +1,28 @@
-<script setup lang="ts">
-  defineEmits<(_type: "update:modelValue", _value: unknown) => void>();
+<script lang="ts">
+  import { defineComponent } from "vue";
+  import { useModelWrapper } from "./helpers";
 
-  defineProps<{
-    disabled?: boolean;
-    invalid?: boolean;
-    modelValue?: string;
-    readonly?: boolean;
-    required?: boolean;
-  }>();
+  export default defineComponent({
+    props: {
+      disabled: { type: Boolean, required: false },
+      invalid: { type: Boolean, required: false },
+      readonly: { type: Boolean, required: false },
+      required: { type: Boolean, required: false },
+      options: { type: [Array, Object], required: true },
+      modelValue: { type: [String, Number, Boolean], require: false, default: "" },
+    },
+    setup(props, { emit }) {
+      return {
+        selected: useModelWrapper(props, emit),
+      };
+    },
+  });
 </script>
 
 <template>
   <select
+    v-if="options && options.length > 0"
+    v-model="selected"
```

Comment:
Do I understand correctly that this code removes the `<select>` when there are no options? I would expect it would just render the `<select>` with no children.

Reply:
Yes, I think having `select` without children is better in case there are no options, but I got `Eslint` [warning](https://eslint.vuejs.org/rules/no-use-v-if-with-v-for.html) Vue preventing the use of ` v-for` directives together with `v-if` directives on the same element.  So, can we have a `.eslintrc` in the `/component-library-vue` to adjust some rules,